pkgs/top-level/aliases.nix

```
@@ -89,6 +89,7 @@ mapAliases ({
   aucdtect = throw "aucdtect: Upstream no longer provides download urls"; # Added 2020-12-26
   avldrums-lv2 = x42-avldrums; # Added 2020-03-29
   avxsynth = throw "avxsynth was removed because it was broken"; # Added 2021-05-18
+  awesome-4-0 = throw "'awesome-4-0' has been renamed to/replaced by 'awesome'"; # Added 2022-03-05
```

```suggestion
  awesome-4-0 = awesome; # Added 2022-03-05
```
we don't need a throw here, since a alias would suffice